### PR TITLE
Add user invitation

### PR DESCRIPTION
**Pull request by psmir**
https://trello.com/c/5tVdI7wQ/14-create-user-by-organization-admin-and-add-him-to-organization

### CODE REVIEW

File: app/mailers/invitation_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class InvitationMailer < ActionMailer::Base
+  def new_invitation(invitation)
+    @invitation = invitation
+    @member = invitation.member
+    @organization = @member.organization
+
+    mail(to: @invitation.email, subject: 'Invitation', from: '<no-reply@cashbox.dev>')
```
Comment (AlexanderUlitin) [inline app/mailers/invitation_mailer.rb]: @Gataniel please use default :from, like this:

```
default from: '<no-reply@cashbox.metastudiohq.com>'
```

Comment (AlexanderUlitin) [inline app/mailers/invitation_mailer.rb]: @Gataniel or move i t to config, like here: http://stackoverflow.com/questions/2445100/actionmailer-default-from-address

File: app/models/member.rb [hunk 2]
```diff
@@ -16,6 +16,8 @@ class Member < ActiveRecord::Base
   belongs_to :user
   belongs_to :organization
 
+  has_many :invitations, foreign_key: :invited_by_id
```
Comment (shir) [inline app/models/member.rb]: @Gataniel rename this association to something like `created_invitations` to differ from `accepted_invitations`

File: app/views/invitation_mailer/new_invitation.html.slim [hunk 3]
```diff
@@ -0,0 +1,4 @@
+= "You are invited to #{@organization.name} as #{@invitation.role}"
```
Comment (shir) [inline app/views/invitation_mailer/new_invitation.html.slim]: @Gataniel add html escaping `"You are invited to #{h @organization.name} as #{h @invitation.role}"`

File: app/views/invitation_mailer/new_invitation.text.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<%= "You are invited to #{@organization.name} as #{@invitation.role}" %>
```
Comment (shir) [inline app/views/invitation_mailer/new_invitation.text.erb]: @Gataniel same here. Use html escaping everywhere where you put data in string.

File: db/seeds.rb [hunk 5]
```diff
@@ -12,3 +20,32 @@
 
 Category.create!(Category::CATEGORY_BANK_EXPENSE_PARAMS)
 Category.create!(Category::CATEGORY_BANK_INCOME_PARAMS)
+
+user = User.create( email: "lukeiam@your.father",
```
Comment (shir) [inline db/seeds.rb]: @Gataniel remove test data from seeds.

**Pull request merged**